README.md

```diff
@@ -131,6 +131,19 @@ interesting are:
   $ cd -
   $ grunt alloy-css
   ```
+  
+* To test updating custom AlloyEditor skin against custom platform-ui-assets-bundle builds:
+
+  ```bash
+  $ composer install --prefer-sopurce
+  $ cd vendor/ezsystems/platform-ui-assets-bundle/
+  $ git checkout <your_branch>
+  $ ./bin/prepare_release.sh -v X.Y.X
+  $ git checkout vX.Y.X
+  $ echo -e `cd Resources/public/vendors/alloy-editor/ && npm install`
```

Comment:
BTW, while I remember it: be aware these instructions will fail if `<your_branch>` contains changes to for instance `package.json` because `./bin/prepare_release.sh` always swaps to master and builds from it. It didn't cause problems for me as the script was already loaded when this was done, but for changes in `package.json` it would.